prow/istio-presubmit.sh

```diff
@@ -113,5 +113,8 @@ if [[ -n $(git diff) ]]; then
   git diff
 fi
 
+# run security e2e test
+${ROOT}/security/bin/e2e.sh --hub "gcr.io/istio-testing" --tag "${GIT_SHA}"
```

Comment:
I said it wrong, we need to put this after `make push`

Reply:
Done.